app/grandchallenge/products/templates/products/base.html

```diff
@@ -19,6 +19,15 @@
     </style>
 {% endblock %}
 
+{% block meta_social %}
+    <meta name="title" content="AI for Radiology">
+    <meta property="og:title" content="AI for Radiology" />
+    <meta property="og:url" content="https://www.aiforradiology.com" />
+    <meta property="og:image" content= "{% static 'products/images/logo-collage-aiforradiology.JPG' %}" />
+    <meta property="og:description" content="A platform to help you find the right AI software for radiology" />
+    {{ block.super }}
```

Comment:
I don't think that you need `{{ block.super }}` in any of these 3 places now that you have split out the other meta params. If you keep it then the properties defined in the super block will override the ones you define here.

Reply:
Sorry if I caused any confusion from my first review!

Reply:
Oh, my understanding was it would complement the items that I have not defined. For example, I do not define the twitter card which is defined in `core/base`, I thought to use it as some sort of fallback mechanism, that if the item is not defined it takes it from `core/base` because of the `{{  block.super }}`. But if it overwrites it I should indeed leave it out. 
https://docs.djangoproject.com/en/3.1/howto/overriding-templates/#extending-an-overridden-template

Reply:
The key point in the docs is the 3rd there:

> The template replaces just the branding block, adding a custom logo, and using block.super to retain the prior content.

I think what is missing from the docs is that the prior content doesn't contain an image already. If it did, you would be adding 2nd image before the 1st included in block.super.

Unfortunately I don't know a way of doing property merging in templates like you'd like, but it would be possible with a python dictionary in a context variable.

Reply:
Thanks for explaining!